### PR TITLE
Timeline component

### PR DESCRIPTION
**Pull request by zachstence**
https://user-images.githubusercontent.com/31896377/211215284-d2b0539a-cc3b-4230-865c-c4547ce35643.mov

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-2q4p9mm Timeline Atoms & Molecules](https://app.clickup.com/t/2q4p9mm)

**Review by gankoji — APPROVED** [review]

**Review by 0c370t — DISMISSED** [review]
Code is good; circles are not centered

File: clients/web/src/lib/components/molecules/Timeline/Timeline.story.svelte [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+    import type {Hst as _Hst} from "@histoire/plugin-svelte";
```
Comment (bwpikaard) [inline clients/web/src/lib/components/molecules/Timeline/Timeline.story.svelte:2]: Formatting?

**Review by bwpikaard — CHANGES_REQUESTED** [review]

File: clients/web/src/lib/components/molecules/Timeline/Timeline.story.svelte (hunk 1 above)
Comment (bwpikaard) [reply clients/web/src/lib/components/molecules/Timeline/Timeline.story.svelte:2]: (Should have no space in brackets in imports)

**Review by 0c370t — APPROVED** [review]

**Pull request merged**